### ISSUE TITLE
Обновить документацию

### ISSUE THREAD

**Issue by dpolyakov** [enb/enb#443]
Если не ошибаюсь, в версии 1 вы выпилили очень много из `enb`, но гайда по переходу я так и не нашел. В Readme до сих пор старые примеры, которые теперь не работают.

**Comment by webhive** [comment]
Поддерживаю. В примерах пути неверные. Как минимум ``enb/techs`` во многих местах надо заменить на ``enb-bem-techs/techs``. Ну и наверно стоит упомянуть, что многое вынесено именно туда.

**Comment by rifler** [comment]
в доке есть технология `deps-old`, но в techs ее нет

**Comment by blond** [comment]
Кто-нибудь хочет помочь? :) Есть предложения?

Думаю, что можно разложить всю документацию на отдельные файлы в папку `docs`.

Всё разом сделать сложно. Можно выбирать одну тему, переносить её в отдельный файл и актуализировать.

**Comment by blond** [comment]
> Если не ошибаюсь, в версии 1 вы выпилили очень много из enb, но гайда по переходу я так и не нашел.

@dpolyakov [История измениний](https://github.com/enb/enb/releases/tag/v1.0.0) не подходит?

Ещё есть гайд по миграции в `enb-bem-techs`: https://github.com/enb/enb-bem-techs/blob/master/MIGRATION-1.md

**Comment by dpolyakov** [comment]
@blond, нет, потому что нужно знать о том, что где-то в другом репозитории есть гайд
